extension/content.css

```diff
@@ -49,12 +49,27 @@
 	display: none !important;
 }
 
-/* Remove tooltips */
-.tooltipped:not(.reaction-summary-item,.avatar-group-item,.avatar):before,
-.tooltipped:not(.reaction-summary-item,.avatar-group-item,.avatar):after {
+/* Remove tooltips where unnecessary */
+.tooltipped:before,
+.tooltipped:after {
 	display: none !important;
 }
 
+.notification-actions .tooltipped:before,
+.notification-actions .tooltipped:after,
+.reaction-summary-item:before,
+.reaction-summary-item:after,
+.avatar-group-item:before,
+.avatar-group-item:after,
+.js-zeroclipboard:before,
+.js-zeroclipboard:after,
+.rgh-tooltipped:before,
+.rgh-tooltipped:after,
+.avatar:before,
+.avatar:after {
+	display: inline-block !important;
+}
```

Comment:
Sadly:

- `:matches` isn't available
- CSS Nesting module isn't available
- `:not(a):not(b)` can't be broken into multiple lines
- `:not(a, b)` isn't available

Any of them would shorten these selectors considerably.

Reply:
I think you can use `:-webkit-any(a, b)` instead of `:matches` for now.

See: https://css-tricks.com/almanac/selectors/m/matches/

Reply:
Bug, Firefox.

Reply:
`:-moz-any()` https://developer.mozilla.org/en-US/docs/Web/CSS/%3Aany

Reply:
Yes. That requires two separate rules though, so we solve nothing 😁